src/main_thread/tracks_store/tracks_store.ts

```diff
@@ -492,6 +528,119 @@ export default class TracksStore extends EventEmitter<ITracksStoreEvents> {
     }
   }
 
+  /**
+   * Handle the noPlayableRepresentation event, trigger an error if no fallback is possible.
+   * and can trigger event "noPlayableTrack"
+   * @param period - The period that has no playable representation
+   * @param bufferType - The media type that is not playable
+   */
+  private onNoPlayableRepresentation(period: IPeriodMetadata, bufferType: IBufferType) {
```

Comment:
Why aren't we using `ITrackType` instead of `IBufferType` here, it seems more semantically appropriate no? (we mean the type of an API track not the type of a SegmentSink)

Reply:
I have changed it, the file uses variable names such `bufferType` or `trackType` so I was not sure which one to use.